### PR TITLE
Unexport internal members of client.LogVerifier.

### PR DESCRIPTION
**Pull request by pphaneuf**

### CODE REVIEW

File: client/log_verifier.go [hunk 1]
```diff
@@ -34,22 +34,22 @@ import (
...
-	// SigHash computes the digest of LogRoot for signing.
-	SigHash crypto.Hash
-	v       logverifier.LogVerifier
+	// hasher is the hash strategy used to compute nodes in the Merkle tree.
+	hasher hashers.LogHasher
+	// pubKey verifies the signature on the digest of LogRoot.
+	pubKey crypto.PublicKey
+	// sigHash computes the digest of LogRoot for signing.
+	sigHash crypto.Hash
+	verifier       logverifier.LogVerifier
```
Comment (pavelkalinnikov) [inline client/log_verifier.go:43]: Personally, I would leave it as `v`. **And** maybe even made other private names 1-2 letters long. Feels like Go style to me.

**Review by pavelkalinnikov — APPROVED** [review]
LGTM % optional nit.

File: client/log_verifier.go (hunk 1 above)
Comment (pphaneuf) [reply client/log_verifier.go]: I find the single-letter names difficult to search for when using more "basic" editors like vi, but done (the important part was unexporting!).

Comment (pavelkalinnikov) [reply client/log_verifier.go]: Good point about vi, didn't look at it from this angle. My approach is usually typing something like `c\.v` to search, but it requires knowing the receiver name (which is hopefully the same across the entire file).

**Pull request merged**